flow/FileTraceLogWriter.cpp

```diff
@@ -87,7 +87,18 @@ void FileTraceLogWriter::write(const std::string& str) {
 void FileTraceLogWriter::open() {
 	cleanupTraceFiles();
 
-	auto finalname = format("%s.%d.%s", basename.c_str(), ++index, extension.c_str());
+	++index;
+	int indexWidth = -1;
+	while (index > 0) {
+		index /= 10;
+		++indexWidth;
+	}
+
+	// this allows one process to write 10 billion log files
+	// this should be enough - if not we could make the base larger...
```

Comment:
Makes sense. We could use 'A', 'B', 'C', ... after '9' for example

Reply:
my initial proposal was to use `0-9`, `a-z`, and `A-Z`. But I was made fun of for assuming an alphabet of size 62 will ever be needed...